### DISCUSSION TITLE
What happened to release v0.50.1?

### DISCUSSION THREAD

**Discussion by somethingvague** [open-policy-agent/community#363] [Community]
The tag is still present https://github.com/open-policy-agent/community/discussions/356 but not backed by a release anymore https://github.com/open-policy-agent/opa/releases?

Context - we automatically upgrade dependencies based on GitHub tags

**Comment by srenatus** [comment]
@johanfylling is working on it right now

**Reply by somethingvague** [reply]
Thank you

**Comment by johanfylling** [comment]
The tag simply went up before the release was published. It should be public now: https://github.com/open-policy-agent/opa/releases/tag/v0.50.1

**Reply by somethingvague** [reply]
Thanks
